etrago/cluster/gas.py

```diff
@@ -51,21 +51,21 @@ def preprocessing(etrago):
     # Cluster ch4 buses
     kmean_settings = etrago.args["network_clustering"]
 
-    ch4_filter = (network_ch4.buses["carrier"].values == "CH4")
+    ch4_filter = network_ch4.buses["carrier"].values == "CH4"
 
     num_neighboring_country = (
-        ch4_filter
-        & (network_ch4.buses["country"] != "DE")
+        ch4_filter & (network_ch4.buses["country"] != "DE")
     ).sum()
 
     # select buses dependent on whether they should be clustered in (only DE or DE+foreign)
     if not kmean_settings["cluster_foreign_gas"]:
 
         network_ch4.buses = network_ch4.buses.loc[
-            ch4_filter
-            & (network_ch4.buses["country"].values == "DE")
+            ch4_filter & (network_ch4.buses["country"].values == "DE")
         ]
 
+        network_ch4.buses = network_ch4.buses.loc[~network_ch4.buses.index.isin(['48627', '48630'])]

```

Comment:
Would it be a problem to cluster these buses to another country as the kmedoids-Dijkstra algorithm suggests?